tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -112,7 +117,7 @@ public function testPostCreate() {
 			->method('insert')
 			->with($this->imageDimension);
 
-		$this->imagehooks->post_create($jpgFile);
+		$this->mockImageHooks->postCreate($jpgFile);
```

Comment:
This is the error written in syslog: `Jun 30 23:54:13 imjalpreet kernel: [ 1406.717116] php[4468]: segfault at 7fff78dc6ff8 ip 000000000079a1e6 sp 00007fff78dc7000 error 6 in php5[400000+7f4000]` but there isn't any php log file in `/var/log/`